src/main/resources/org/jvnet/hudson/plugins/collapsingconsolesections/CollapsingSectionNote/script.js

```diff
@@ -95,70 +95,89 @@ function doToggle(o)
         return true;
     }
 
-    function generateOutlineSection($sectionElt){
+    function generateOutlineSection(sectionElt){
 
         var id = "console-section-"+(iota++);
     	// add target link in output log
-        var $a =jQuery("<a name='"+id+"' />");
-        $a.prependTo($sectionElt);  
+        var a = document.createElement("a");
+        a.name = id;
+        sectionElt.prepend(a);  
     	
     	// create outline element
-    	var $collapseHeader = $sectionElt.children("DIV.collapseHeader")
-        var $elt = jQuery("<li/>")
-        $elt.append(jQuery("<a href= '#"+id+"'>"+justtext($collapseHeader)+"</a>"));
+    	var collapseHeader = sectionElt.querySelector("DIV.collapseHeader")
+        var elt = document.createElement("li")
```

Comment:
Those two lines are missing semicolons at the end, I also noticed multiple lines are indented with tabulations when the file uses 4 spaces for indentation. Ideally the repository would have ESLint configured and a GitHub action to catch it :-D

Reply:
**solved**
I am following the guide at https://github.com/jenkinsci/plugin-pom/  to integrate ESLint in Maven/CI and will send another pull request for that.